librz/include/rz_bin.h

```diff
@@ -459,6 +459,8 @@ typedef struct rz_bin_plugin_t {
 	int (*get_offset)(RzBinFile *bf, int type, int idx);
 	char *(*get_name)(RzBinFile *bf, int type, int idx, bool simplified);
 	ut64 (*get_vaddr)(RzBinFile *bf, ut64 baddr, ut64 paddr, ut64 vaddr);
+	char* (*section_type_to_string)(ut64 type);
+	char* (*section_flag_to_string)(ut64 flag);
```

Comment:
```suggestion
	RzList *(*section_flags_to_string)(ut64 flag);
```
I propose two changes:
1) flags instead of flag, because a section can have more than one flag.
2) return a RzList instead of just a string. This would be useful e.g. when printing the flags in JSON mode, where you can print the list of flags instead of just a string representing all of them.

Reply:
It would also avoid the problem of having the space after each flag, because only when you print them you would add a space in-between two flags.